crates/biome_js_analyze/tests/specs/nursery/useExplicitTestAssertions/invalid.js.snap

```diff
@@ -0,0 +1,144 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+expression: invalid.js
+---
+# Input
+```js
+it('should fail', () => {});
+
+it('should fail', myTest);
+function myTest() {}
+
+test('should fail', () => {});
+
+test.skip('should fail', () => {});
+
+it('should fail', () => {
+	somePromise.then(() => {});
+});
+
+test('should fail', () => {
+	foo(true).toBe(true);
+});
+
+it('should also fail', () => expectSaga(mySaga).returns());
+
+```
+
+# Diagnostics
+```
+invalid.js:1:1 lint/nursery/useExplicitTestAssertions ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i Missing assertion in test body.
+  
+  > 1 │ it('should fail', () => {});
```

Comment:
🙏 Please double-check these to make sure they’re correct! I know it’s _mostly_ correct but needs a better eye